tests/recipes/test_reportlab.py

```diff
@@ -0,0 +1,54 @@
+import os
+import unittest
+from mock import patch
+from pythonforandroid.archs import ArchARMv7_a
+from pythonforandroid.build import Context
+from pythonforandroid.graph import get_recipe_order_and_bootstrap
+from pythonforandroid.recipe import Recipe
+from pythonforandroid.util import ensure_dir
+
+
+class TestReportLabRecipe(unittest.TestCase):
+
+    def setUp(self):
+        """
+        Setups recipe and context.
+        """
+        self.context = Context()
+        self.arch = ArchARMv7_a(self.context)
+        self.recipe = Recipe.get_recipe('reportlab', self.context)
+        self.recipe.ctx = self.context
+        self.bootstrap = None
+        recipe_build_order, python_modules, bootstrap = \
+            get_recipe_order_and_bootstrap(
+                self.context, [self.recipe.name], self.bootstrap)
+        self.context.recipe_build_order = recipe_build_order
+        self.context.python_modules = python_modules
+        self.context.setup_dirs('/tmp/')
```

Comment:
Not sure how portable we need to be here, but should probably use `tempfile.gettempdir()` instead. or even tempfile.TemporaryDirectory if we want/need to be isolated from other runs.

Reply:
Addressed, thanks!